pkg/volume/glusterfs/glusterfs.go

```diff
@@ -135,6 +135,10 @@ type glusterfsBuilder struct {
 
 var _ volume.Builder = &glusterfsBuilder{}
 
+func (_ *glusterfsBuilder) SupportsOwnershipManagement() bool {
```

Comment:
Glusterfs mount is based on FUSE. Potentially, we can make use of FUSE allow_other to allow non-root users to access files.